src/components/Layout/ScrollableContent.tsx

```diff
@@ -28,18 +28,15 @@ export function ScrollableContent(props: ScrollableContentProps): ReactPortal |
   }
 
   return createPortal(
-    !virtualized ? (
-      omitBottomPadding ? (
-        children
-      ) : (
-        <div css={scrollContainerBottomPadding}>{children}</div>
-      )
-    ) : (
-      // To prevent Virtuoso's scrollbar from being set in based on the Layout's padding, we will use the FullBleed component w/o padding to push it back over
-      <FullBleed omitPadding>
-        <div css={Css.h100.pl(pl).$}>{children}</div>
-      </FullBleed>
-    ),
+    <div
+      css={{
+        ...Css.pr(pr).pl(pl).if(virtualized).pr0.h100.$,
+        ...(bgColor && Css.bgColor(bgColor).$),
+        ...(!omitBottomPadding && !virtualized && scrollContainerBottomPadding),
+      }}
+    >
+      {children}
+    </div>,
```

Comment:
Moving the `paddingLeft` and `paddingRight` definitions here allows us to remove the need of the `<FullBleed />` component when the `virtualized` prop is `true`.  This helped simplify the return logic as well.